packages/rulesets/src/spectral/az-dataplane.ts

```diff
@@ -495,6 +496,17 @@ const ruleset: any = {
         function: hostParameters,
       },
     },
+    LongRunningResponseStatusCodeDataPlane: {
```

Comment:
Adding a doc for this dataplane only rule, as the original rule calls LongRunningResponseStatusCode